_tour/basics.md

```diff
@@ -100,17 +97,17 @@ println({
 
 ## Functions
 
-Functions are expressions that take parameters.
+Functions are expressions that have parameters, and take arguments.
```

Comment:
This is redundant.  Having parameters is equivalent to taking arguments.  The original is better.  (Also, functions can take zero arguments, so the statement isn't even correct either way, but the new one is worse than the old one.)

Also, although "arguments" is more standard terminology, the parameters are called parameters in Scala, so it's probably better to leave it alone.

Reply:
I take your point about the functions having zero arguments, but the original statement saying a function can "take" parameters doesn't make sense either.

 A function has parameters. When it says functions "take" parameters it sounds like "parameters" can be passed in. Arguments can be passed in, parameters can't.

Reply:
Fair enough, but I don't think the clarity of the text should rely upon the reader understanding that arguments are the passed-in version of parameters.

Reply:
If a reader sees "functions can _take_ parameters", they might wonder about the quality, and accuracy of the Scala documentation as a whole.

The information should be made as accurate as possible, regardless of the reader's level of understanding. 

I think this a (living) language thing. Some languages call them two different types of parameters; English calls them parameters and arguments.

Reply:
I'm going to merge the PR regardless, but I lean more towards Rex's side here. I invite anyone interested to submit a followup PR with a different wording that we all like?